G2P.pm

```diff
@@ -480,18 +492,20 @@ sub get_header_info {
 =cut
 sub run {
   my ($self, $tva, $line) = @_;
-
+  
   # only interested if we know the zygosity
   my $zyg = defined($line->{Extra}) ? $line->{Extra}->{ZYG} : $line->{ZYG};
   return {} unless $zyg;
+
+  return {} if ($self->{user_params}->{only_mane} == 1 && !$tva->transcript->is_mane);
```

Comment:
If the option `only_mane` is not defined, VEP throws a warning `Use of uninitialized value in numeric eq (==)`.

You can change the condition to:
`if ($self->{user_params}->{only_mane} && !$tva->transcript->is_mane);`